spec/support/features/todo_helper.rb

```diff
@@ -5,11 +5,20 @@ def create_todo(todo_title)
         click_on "Submit"
     end
 
+    def display_li()
```

Comment:
We don't need parenthesis at the end of a method call unless you are passing in arguments